components/script/dom/document_embedder_controls.rs

```diff
@@ -280,7 +290,14 @@ impl DocumentEmbedderControls {
             ]);
         }
 
-        if image_element.is_some() {
+        if let Some(image_element) = image_element.as_ref() {
+            info.flags.insert(ContextMenuElementInformationFlags::Image);
+            info.image_url = document
```

Comment:
Can you expose a method to return https://github.com/servo/servo/blob/8e0c2d57504b88ec145198d6821ebfb88e3e5c71/components/script/dom/html/htmlimageelement.rs#L1526 instead of using Src()? That will make responsive images less surprising.